.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -18,17 +43,10 @@ Steps to reproduce the behavior:
 4. Scroll down to '....'
 5. See error
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
-
-**Screenshots**
-If applicable, add screenshots or a screen recording to help explain your problem.
-
 **Smartphone (please complete the following information):**
- - Device: [e.g. Saga, Pixel 7, etc.]
+ - Device: [e.g. Seeker, Pixel 7, etc.]
  - Browser: if this issue is related to mobile web [e.g. Chrome, Firefox]
- - Version: if this issue is endemic to specific Android versions [e.g. 22]
```

Comment:
why was android version removed? I guess we never see stuff related to android version but tis certainly possible. especially as we roll out updates and have Seekers on different android versions